core/src/main/java/org/elasticsearch/plugins/PluginManager.java

```diff
@@ -582,17 +583,24 @@ public static PrintStream getOut() {
         List<URL> urls() {
             List<URL> urls = new ArrayList<>();
             if (version != null) {
-                // Elasticsearch download service
-                addUrl(urls, "http://download.elasticsearch.org/" + user + "/" + repo + "/" + repo + "-" + version + ".zip");
-                // Maven central repository
-                addUrl(urls, "http://search.maven.org/remotecontent?filepath=" + user.replace('.', '/') + "/" + repo + "/" + version + "/" + repo + "-" + version + ".zip");
-                // Sonatype repository
-                addUrl(urls, "https://oss.sonatype.org/service/local/repositories/releases/content/" + user.replace('.', '/') + "/" + repo + "/" + version + "/" + repo + "-" + version + ".zip");
-                // Github repository
-                addUrl(urls, "https://github.com/" + user + "/" + repo + "/archive/" + version + ".zip");
+                // Elasticsearch new download service uses groupId org.elasticsearch.plugins from 2.0.0
+                if (user == null) {
+                    addUrl(urls, String.format("https://download.elastic.co/org.elasticsearch.plugins/%1$s/%1$s-%2$s.zip", repo, version));
```

Comment:
have you run this through maven? I think `String.format()` without a locale is part of the forbidden API